README.md

```diff
@@ -116,6 +118,11 @@ After installing this npm package, add the [config plugin](https://docs.expo.io/
 ```json
 {
   "expo": {
+    ios: {
+      infoPlist: {
```

Comment:
This seems incorrect json.
Also, what about apps that does not use expo?

Reply:
@DavideViolante should he wrap the keys i.e.(`ios` and `infoPlist`) in quotes to push this through? Rather than also address this for all React Native users, what if he includes a header with something like "For Expo Projects"?